src/NuGet.Core/NuGet.PackageManagement/Strings.resx

```diff
@@ -332,4 +332,7 @@
     <value>Source '{0}' not found. Please provide an HTTP or local source.</value>
     <comment>{0} is the source</comment>
   </data>
+  <data name="NoPackageUpdates" xml:space="preserve">
+    <value>No package updates are available from the current package source for project '{0}'.</value>
```

Comment:
This is the existing message which is being shown currently and more over user already knows about what what source is being selected... but I can add sources if that's what we want!